atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionManager.java

```diff
@@ -53,7 +53,7 @@
                 if (!e.canTransactionBeRetried()) {
                     log.warn("[{}] Non-retriable exception while processing transaction.",
                             SafeArg.of("runId", runId),
-                            SafeArg.of("failureCount", failureCount));
+                            e);
```

Comment:
Do we want to log the stacktrace here if we're rethrowing? You might want to log the simple classname but not stacktrace.